lib/registration/registration_message.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:bloc_mvu_app/registration/registration_update.dart';
+import 'package:bloc_mvu_app/mvu/messaging.dart';
+
+abstract class RegistrationMessage extends UpdateMessage {
+  RegistrationMessage() : super(routeTo: RegistrationUpdate);
+}
+
+class changePasswordVisibilityMsg extends RegistrationMessage {}
```

Comment:
pls start class names in Uppercase. Please get rid of the superfluous "Msg"-suffix (smurf naming)